src/main/java/org/bytecodeparser/instruction/Instruction.java

```diff
@@ -2,11 +2,29 @@
 
 import org.bytecodeparser.print.CustomPrettyPrint;
 
+import java.io.DataInputStream;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
 @CustomPrettyPrint
-public interface Instruction {
+public class Instruction {
+
+    private static final Map<Integer, InstructionTypes> instructions = Arrays.stream(InstructionTypes.values())
```

Comment:
EnumSet is designed to be efficient, it is best practice to use it, not values()

Reply:
It is not agree with u: https://stackoverflow.com/questions/2464950/enum-values-vs-enumset-allof-which-one-is-more-preferable

Reply:
But thx! before I don't know about EnumSet)

Reply:
values is method which is generated by compiler and aims for optimiozation, EnumSet.allOf() uses it under the hood. There are no doubts in performance, it is about the conventional way how to work with collections of enum constants

Reply:
ok, +